### ISSUE TITLE
images in the category dropdown do not have alt-attributes

### ISSUE THREAD

**Issue by fuzzylogic2000** [liqd/a4-meinberlin#1564]
At least they don't in the component library. We need to check if they have (empty) alt-attributes in the real things, so that a screen reader doesn't read out the filename! Might be in A4, though.

**Comment by phillimorland** [comment]
@phillimorland to check all images to check they have alt attributes 
pins images needs investigated - how to treat diff? how it gets in there?
